db/migrate/20210730191314_create_ecf_participant_eligibility.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateECFParticipantEligibility < ActiveRecord::Migration[6.1]
+  def change
+    create_table :ecf_participant_eligibilities, id: :uuid do |t|
+      t.references :participant_profile, index: true, foreign_key: true
+      t.boolean :qts
+      t.boolean :active_flags
+      t.boolean :previous_participation
+      t.boolean :previous_induction
+      t.string :status, null: false, default: "manual_check"
```

Comment:
Why persist this? Surely this is purely calculated?

Reply:
Simplify checking/querying with the enum sugar and scopes basically. Also allows us to do the holding pen thing easily by setting the status independently of the flags in `#determine_status`.